wordpress-mobile/WordPress-iOS
# Stats Revamp: Likes Details Screen

This PR adds the Details (2nd level) Likes screen

Fixes #18603

Note:
 * View More disclosure on Details screen doesn't do anything yet + will be handled in future PR

Design
<img width="300" alt="image" src="https://user-images.githubusercontent.com/88816724/173063293-f13ce6e9-04c7-4e86-82c0-e21f79e37189.png">


Implementation
<img width="300" alt="image" src="https://user-images.githubusercontent.com/88816724/173063105-33efd7b5-914d-476c-947d-5255ca18f45b.png">


To test:
1. Build and run + navigate to Stats Insights for a site
2. Likes Total card should not be available
3. Click around insights, period stats and ensure there are no crashes
4. Now enable both of the new stats feature flags and build and run again
        * statsNewAppearance
        * statsNewInsights
5. From the top level Stats Insights screen add the following cards: Likes Total, Followers Total, Comments Total if not already visible.  All 3 of these cards should look exactly the same as before this PR (note that I moved makeSparklineData into StatsTotalInsightCell so that the logic could be shared between top level insights and 2nd level insights so this is to test there is no unintended impact)
6. On the top level Stats Insights screen for cards Comments Total and Followers Total - tap the right chevron which should navigate you to the details screen + there should be no change in functionality compared to before this PR
7. On the top level Stats Insights screen scroll to Likes Total and tap the right chevron
8. You should see the Likes Total card followed by the Post and Pages card
9. Tap back and forth on the DatePicker at the top + Like Totals and Posts and Pages should update accordingly

## Regression Notes
1. Potential unintended areas of impact
Stats screens

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
